packages/wallet/Account.js

```diff
@@ -0,0 +1,604 @@
+const rp = require('request-promise');
+const StellarSdk = require('stellar-sdk');
+const Web3 = require('web3');
+const openurl = require('openurl');
+const toml = require('toml');
+
+const server = new StellarSdk.Server('https://horizon-testnet.stellar.org');
+const web3 = new Web3('https://rinkeby.infura.io/v3/54add33f289d4856968099c7dff630a7');
+
+/**
+ * @class Account
+ * This is a wrapper class over stellar and ethereum accounts.
+ * For those fields/methods that are already there when the original
+ * account is passed in, this class simply extracts them;
+ * Otherwise, the fields/methods are created and saved in this class.
+ */
+class Account {
+  /**
+  * constructor
+  * @param {string} network
+  */
+  constructor(network) {
+    switch (network) {
+      case 'stellar':
+        this.network = 'stellar';
+        this.testAddress = 'https://horizon-testnet.stellar.org/accounts/';
+        StellarSdk.Network.useTestNetwork();
+        break;
+      case 'ethereum':
+        this.network = 'ethereum';
+        break;
+      default:
+        console.log('The name of the network must be stellar or ethereum.');
+    }
+  }
+
+  /**
+   * This is for stellar only
+   * @param {string} assetName
+   * @param {string} issuerPublickey
+   * @param {string} limit
+   * @returns {object|null} xdr.ChangeTrustOp if succeeds or null otherwise
+   */
+  async changeTrust(assetName, issuerPublickey, limit) {
+    switch (this.network) {
+      case 'stellar': {
+        const newAsset = new StellarSdk.Asset(assetName, issuerPublickey);
+        const receiver = await server.loadAccount(this.getAddress());
+        const transaction = new StellarSdk.TransactionBuilder(receiver)
+          .addOperation(StellarSdk.Operation.changeTrust({
+            asset: newAsset,
+            limit
+          }))
+          .build();
+        transaction.sign(this.account);
+        return server.submitTransaction(transaction);
+      }
+      case 'ethereum':
+        console.log('Change trust is not for ethereum.');
+        break;
+      default:
+        console.log('The network is not set correctly.');
+    }
+    return null;
+  }
+
+  /**
+  * set the balance field of this account
+  * @param {string} publicKey - the publick key of the account
+  */
+  async loadBalance(publicKey) {
+    try {
+      const response = await rp(this.testAddress + publicKey);
+      this.balance = JSON.parse(response).balances.slice(-1)[0].balance;// XLM balance
+    } catch (err) {
+      console.log('Error in loading balance');
+    }
+  }
+
+  /**
+   * Set the account field of this account to be the argument;
+   * For Stellar, upload it to the testnet and update the balance;
+   * For Ethereum, update its balance read on the Rinbeky testnet.
+   * @param {object} account - A stellar or ethereum account
+   */
+  async setAccount(originalAccount) {
+    switch (this.network) {
+      case 'stellar':
+        this.account = originalAccount;
```

Comment:
I think we need more tighter scope variable name than `account` for `this.account` in `Account` and `WalletManager`. It is quite confusing when trying to figure out the exact nature of `this.account` - is it `Account` class? Web3 account? Stellar SDK account?

It doesn't help that some `Account` methods has same name with SDK account class e.g `sign`. Perhaps for `Account`, use `this.nativeAccount` instead?

Reply:
also, check the account passed in

Reply:
use this.nativeAccout, can use account as parameter name